sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -54,29 +80,48 @@ public bool SaveDebugRecordingsOnFailure
             }
         }
         private bool _saveDebugRecordingsOnFailure;
+        private Process _testProxyProcess;
+        private static readonly string s_dotNetExe;
```

Comment:
how many dotnet exes do we have :)

Reply:
it depends on the platform.

Reply:
I mean variables, we figure out the whole path and store it into a different variable. Do we need this one?

Reply:
yeah it is used in multiple places

Reply:
Why aren't we using the full path there?

Reply:
it is used in the method that check to see if the dotnet exe is in the path.